### DISCUSSION TITLE
关于魔神战争

### DISCUSSION THREAD

**Discussion by nahida0-o** [share121/inter-knot#450] [其他]
昔日的盐之魔神曾带领子民躲到了地中之盐这偏僻的地方，但仍未逃得一死，也没有成功为子民们谋得祂所期待的幸福。那么如今，在这里，我是否可以畅所欲言而不需要畏惧无尽的战火（或许九月就会结束？），与正常的人一起讨论而非面对无尽的智械？
![aad6ebc5248c38f2806fd89339d55b1](https://github.com/user-attachments/assets/1f7bc3c0-39c2-42a7-8fb8-409be51a8247)

**Comment by BSNHD** [comment]
至少现在肯定是没问题的，顺便问问你是妲厨吗[派蒙_流口水]

**Reply by nahida0-o** [reply]
肯定的，不过在展开讨论之前我还有点事要办

**Reply by nahida0-o** [reply]
当然是妲厨！（不过其实更偏向谁都推一把的单推人）

**Reply by nahida0-o** [reply]
谁对我好（我抽谁抽的顺利）我就推谁，纳西妲的一二命都是在我觉得不可能出的时候十几抽就出了，所以我是妲厨。[图片]

**Reply by nahida0-o** [reply]
现在倒也不完全按照抽卡情况来判断了，比如藿藿，也不怎么欧，但就是挺喜欢的又能怎么办呢

**Reply by nahida0-o** [reply]
我突然发现一件事，就是为什么我喜欢的大多都身材较为娇小，表现比较可爱......难道，我真是.......

**Reply by BSNHD** [reply]
没事，现实和虚拟区分开就行，别太在意[ok]

**Comment by Xmug** [comment]
![纳西妲验孕棒 爱情可以跨越年龄](https://github.com/user-attachments/assets/9a5ebc36-637d-4460-9853-f9b15de59379)

**Reply by nahida0-o** [reply]
刑警也能跨过我家大门。
不过如果他们能坐实证据也好啊，起码那样我就真的拥有了一个纳西妲，而不用这样每天只是之于无尽的幻想。

**Comment by nahida0-o** [comment]
其实，琦良良有个外号叫“妙哉仙人”。因为她老板每次一喊她她就回答“喵在，喵在”（妙哉，妙哉）。
很好笑吧？至少我这么觉得，大风”机关“独特的品味总是想让人忍不住去看他的下一个冷笑话是什么。
![d3c2f2d9719b2f2335963a4c522ec17](ht
![306c3aae0cc9a01993cf1560eebd58a](https://github.com/user-attachments/assets/447e810d-9328-4ef9-83f2-c1be4e5e0376)
tps://github.com/user-attachments/assets/614800ae-5dbd-4ffd-8ead-b249e2347f66)
![5e407682bb1a5e683d0bfd3a2cc1346](https://github.com/user-attachments/assets/bef5230c-e2e5-4c0e-afa1-676f3ce56f44)
![c6adf62e3f569e6baba2b3e8bd04b83](https://github.com/user-attachments/assets/44424b24-56c9-4fa4-9a02-44e76a25c2c5)
![8ad388647db2b3cc59ad52224734204](https://github.com/user-attachments/assets/faaf0617-1663-4558-b462-4c26d5a9ba23)

**Reply by nahida0-o** [reply]
以及
![6a3234e5c7f5b22a46c993068f72da2](https://github.com/user-attachments/assets/b9e374cb-5ffd-4ee7-98ca-19328cfd909b)
找的好累。

**Comment by Xmug** [comment]
![算我求求你，不要再说谐音梗冷笑话了，你很想用这种方式在异国他乡出名吗？](https://github.com/user-attachments/assets/3df74400-7ddb-4d09-8326-f396126c98bb)

**Comment by trv3wood** [comment]
确实，此处是难得的一方净土了

**Comment by Lanxey** [comment]
越是小众越是纯粹，于是大众越是混乱。
至少目前来看想要进来还是有一定门槛的，短期内不必担心智械危机，希望作者后续能继续迭代。
